providers/nodeReleasesProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import { createContext } from 'react';
+import { useFetchNodeReleases } from '../hooks/useFetchNodeReleases';
+import type { FC, PropsWithChildren } from 'react';
+import type { NodeRelease } from '../types';
+
+export const NodeReleasesContext = createContext<{
+  releases: NodeRelease[];
+  lts?: NodeRelease;
```

Comment:
I'm wondering if `lts` and `current` should be a prop here.

I feel we should rewrite this hook to return: `{ releases: NodeRelease[], getReleaseByStatus: (status: ReleaseStatus) => NodeRelease | null }`

This means, the `NodeReleasesContext has only `releases` and the `useNodeReleases` would return what I mentioned above.

I feel this is a better and cleaner way of doing this. Note that the `getNodeReleaseByStatus` can also be memoized if needed.